config/crd/bases/apps.emqx.io_emqxbrokers.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.2
```

Comment:
Do not modify this, we need 0.6.2, please update `controller-gen` for you local, you just can delete `bin/controller-gen` in project root path, and rerun 'make manifests'